docs/artifacts/agentsview-fit.html

```
@@ -0,0 +1,136 @@
+<title>agentsview</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)